CMakeLists.txt

```diff
@@ -24,8 +25,10 @@ option_with_print(ENABLE_OPENMP "Enable threadsafe linking to OpenMP parallelize
 
 # <<<  Prepare  >>>
 
+include(custom_color_messages)
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
+include(custom_color_messages)
```

Comment:
why twice?

Reply:
This should have remained from the time I was trying to separate dependencies from Psi4. Since it exists in the cmake folder of the root directory in Psi4, it can probably go away as long as CMAKE_MODULE_PATH variable is properly set.

Reply:
I didn't mean why custom_color_messages is duplicated from psi4 -- you were right to do so. I meant you have `include(custom_color_messages)` as both lines 28 and 31, which is harmless but potentially confusing.

Reply:
Yes. I got your point when I looked at the code carefully before removing the configure script in the new commit and I appreciate your follow up clarification for it. Thanks Lori.